### ISSUE TITLE
Major error not caught by pipeline

### ISSUE THREAD

**Issue by nicolerg** [ENCODE-DCC/atac-seq-pipeline#48]
**Major error not caught by pipeline**  

The pipeline does not output an error in `call-macs2/shard-0/execution/stderr` if the specified blacklist BED file is improperly formatted. It instead runs to completion, which results in empty final peak files without an obvious error.  

The problem occurs within the following step in the `call-macs2` module (from `call-macs2/shard-0/execution/stdout`):  
```bash
[2018-10-28 03:38:52,988 INFO] run_shell_cmd: PID=131102, CMD=bedtools intersect -v -a 20180815-14-Adipose-002-powder_S14_L001_R1_001.trim.merged.nodup.tn5.pval0.01.300K.narrowPeak.tmp1 -b rn6_blacklist.bed.tmp2 | awk 'BEGIN{OFS="\t"} {if ($5>1000) $5=1000; print $0}' | grep -P 'chr[\dXY]+[ \t]' | gzip -nc > 20180815-14-Adipose-002-powder_S14_L001_R1_001.trim.merged.nodup.tn5.pval0.01.300K.bfilt.narrowPeak.gz
```

It is easy to see why an improperly formatted BED file would have caused this step to fail, but it should have output the error to the `stderr` file (which was empty in `call-macs2/shard-0/execution`) and terminated the pipeline. Instead, at first glance it appeared the pipeline finished running without error and took a bit of digging to find the problem.  

I don't have the exact error or log files because they were removed by another user after fixing the issue.  

**OS/Platform and dependencies**  

- Platform: Ubuntu 16.04.4
- Cromwell: cromwell-34
- Conda version: conda 4.5.11

**Comment by leepc12** [comment]
We are working on it. This will be fixed in the next release.

**Comment by leepc12** [comment]
@nicolerg : We found picard bug to be a serious one and made a hot fix for it. You can simply `git pull` from master and re-install conda depdencies.
```
$ bash conda/uninstall_dependencies.sh
$ bash conda/install_dependencies.sh
```

**Comment by leepc12** [comment]
`bedtools` doesn't seem to properly output errors and warnings to STDERR. I added a simple sanity check for output files. This will be fixed in v1.1.2.
